specification/api/components/schemas/LetterStatus.yml

```
@@ -0,0 +1,17 @@
+type: string
+enum:
+  - PENDING
+  - ACCEPTED
+  - REJECTED
+  - PRINTED
+  - ENCLOSED
+  - CANCELLED
+  - DISPATCHED
```

```suggestion
  - DISPATCHED
  - DELIVERED
```
We've added a `DELIVERED` status to the spec as some suppliers are able to track the date a letter is sent out for delivery.